File: ProgramingBasics/Simple-Loops/06.MinNumber/src/MinNumber.java
import java.util.Scanner;

public class MinNumber {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        int min = Integer.MAX_VALUE;

        for (int i = 0; i < n; i++) {
            int num = Integer.parseInt(scanner.nextLine());

            if (num < min) {
                min = num;
            }
        }

        System.out.println(min);
    }
}

File: ProgramingBasics/SimpleLoops/06.MinNumber/06.MinNumber/MinNumber.cs
using System;

namespace _06.MinNumber
{
    public class MinNumber
    {
        public static void Main(string[] args)
        {
            var n = int.Parse(Console.ReadLine());
            var min = int.MaxValue;
            var num = 0;

            for (var i = 0; i < n; i++)
            {
                num = int.Parse(Console.ReadLine());

                if (num < min)
                {
                    min = num;
                }
            }

            Console.WriteLine(min);
        }
    }
}
